modules/300-prometheus/openapi/values.yaml

```diff
@@ -235,6 +235,12 @@ properties:
                   type: string
                 customAuthToken:
                   type: string
+                headers:
+                  type: object
+                  patternProperties:
+                    "^X-Scope-OrgID$": 
+                      type: string
+                  additionalProperties: false
```

Comment:
```suggestion
```

These fields are required exclusively in the remotewrite CRD and should be removed from the module CRD.